clients/src/test/java/org/apache/kafka/clients/producer/MockProducerTest.java

```diff
@@ -152,15 +151,15 @@ public void shouldBeginTransactions() {
     @Test(expected = IllegalStateException.class)
     public void shouldThrowOnSendOffsetsToTransactionIfTransactionsNotInitialized() {
         buildMockProducer(true);
-        producer.sendOffsetsToTransaction(null, null);
+        producer.sendOffsetsToTransaction(null, "");
```

Comment:
Why this change? Should `sendOffsetsToTransaction` not be able to handle `null` gracefully? Seems it would be a regression if we change the behavior and start to fail on `null`?

Reply:
It is actually not, I will try to specify the object type so we don't need to use 0 length string